charts/super-agent/values.yaml

```diff
@@ -94,71 +94,76 @@ super-agent-deployment:
   # @default -- `false`
   nrStaging:
 
-  # -- (bool) Changes default endpoint to point to EU backend.
-  # @default -- `false`
-  euEndpoints:
-
   fedramp:
     # -- (bool) TODO: Enables FedRAMP. Can be configured also with `global.fedramp.enabled`
     # @default -- `false`
     enabled:
 
-  # -- (bool) TODO: Sets the debug logs to this integration or all integrations if it is set globally. Can be configured also with `global.verboseLog`
+  # -- (bool) Sets the debug logs to this integration or all integrations if it is set globally. Can be configured also with `global.verboseLog`
   # @default -- `false`
   verboseLog:
 
   # -- Enable the cleanup of super-agent managed resources when the chart is uninstalled.
-  # If disabled, agents and / or agent configurations managed by the super-agent will not be deleted when the chart is uninstalled.
+  # If disabled, agents and/or agent configurations managed by the super-agent will not be deleted when the chart is uninstalled.
   cleanupManagedResources: true
 
   config:
-    auth:
-      enabled: false
-      organizationId:
-      userApiKey:
-      secret:
-        create: true
-
     # -- Configuration for the Super Agent.
     # @default -- See `values.yaml`
     superAgent:
       # -- Set if the configMap is going to be created by this chart or the user will provide its own.
       create: true
-      # -- Here you can set New Relic' Super Agent configuration.
-      # @default -- See `values.yaml` for examples
-      content:
-        agents:
-          open-telemetry:
-            agent_type: newrelic/io.opentelemetry.collector:0.2.0
-
-        # opamp:
-        #   # TODO The endpoint should be set automatically based on the licenseKey and on the nrStaging option if opamp.enable=true
-        #   endpoint: https://opamp.service.newrelic.com/v1/opamp
-        #   endpoint: https://opamp.service.eu.newrelic.com/v1/opamp
-
-        # -- This option enables a status server that can be useful for troubleshooting.
-        # -- Port-forward it `$ kubectl port-forward pod/{pod-name} 51200:51200`
-        # -- And query it as `$ curl localhost:51200/status`
-        server:
-          enabled: true
-        #   port: 51200
-
+      # -- It you are a PowerUser, you can use this to override the configuration that has been created automatically by the chart.
+      # This configuration here will be **MERGED** with the configuration specified above.
+      # If you need to have you own configuration, disabled the creation of this configMap and create your own.
+      content: {}
 
     # -- Values that the fleet is going to have in the deployment.
-    # @default -- See `values.yaml` for examples
+    # @default -- {} (Empty. That defaults to configure the `newrelic/io.opentelemetry.collector` subagent)
```

Comment:
I think that this is not clear if you do not know what it is going on

Reply:
I think that if we say something: "If empty the chart add automatically the `newrelic/io.opentelemetry.collector` subagent. On the other hand, if populated the list of agent created is the one specified overwriting the default"

Reply:
Done [#f11ce5a](https://github.com/newrelic/helm-charts/pull/1466/commits/f11ce5aa8c6e8b9def8f699df1103b6f11ac4f96). Let me know what you think :)